seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1418,6 +1418,7 @@ def test_validate_callset(self, mock_subprocess, mock_glob, mock_os_isfile):
         response = self.client.post(url, content_type='application/json', data=json.dumps(body))
         self.assertEqual(response.status_code, 200)
 
+    @mock.patch('seqr.views.utils.terra_api_utils.SOCIAL_AUTH_PROVIDER', TEST_OAUTH2_PROVIDER)
```

Comment:
If this is properly mocked in the bas class it should not be need to be added in any of the test files